llpc/test/shaderdb/partial_pipeline_cache_test/pipelines/PipelineVsFs_ConstantData_Vs1Fs2.pipe

```diff
@@ -0,0 +1,66 @@
+; Test that constant data in the vertex shader is handled correctly.
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -enable-relocatable-shader-elf -o %t.elf %gfxip %s
+; RUN: llvm-objdump --arch=amdgcn --disassemble-zeroes --mcpu=gfx900 -D -r %t.elf | FileCheck -check-prefix=SHADERTEST %s
```

Comment:
Same as in the other test

Reply:
It seems that `llvm-objdump` will not generate `AMDLLPC SUCCESS`.